### PR TITLE
Add 3x3x3 Team-Blind icon

### PR DESCRIPTION
**Pull request by dmint789**
This solves issue #100. The design was briefly discussed in issue #105. The convention will now be that one circle represents one person, which other team event icons can also follow. Team-Blind Old Style is intended to be used [here](https://cubingcontests.com/rankings/333tbfo/single). Old Style refers to Team-Blind with inspection, whereas the new format on cubingcontests.com is without it, so I want to use this old style icon for the old format.

Previews:

333tbf

333tbfo

### CODE REVIEW

File: svgs/unofficial/333tbf.svg
Comment (jfly) [inline svgs/unofficial/333tbf.svg:1]: Out of curiosity, why did you decide to turn this upside-down compared to what you've got [on funcubing](https://funcubing.org/competitions/rankings)?

image

**Comment by lgarron** [comment]
I am unfortunately opposed to this — this event is unofficial, so there isn't really an "old" version of the event. I also intend to keep advocating for "across the table" Team BLD to make it more practical to judge, and wouldn't want to see more and more "old" versions of events that aren't even official yet.

File: svgs/unofficial/333tbf.svg
Comment (lgarron) [reply svgs/unofficial/333tbf.svg:1]: Indeed, it's very cute that way!

Comment (dmint789) [reply svgs/unofficial/333tbf.svg:1]: I thought it looked better this way. It kinda bothered me that their version looks like a smiley face. If you guys prefer I'll change it back.

**Comment by dmint789** [comment]
> I am unfortunately opposed to an "old" version — this event is unofficial, so there isn't really an "old" version of the event. I also intend to keep advocating for "across the table" Team BLD to make it more practical to judge, and wouldn't want to see more and more "old" versions of events that aren't even official yet.

I understand, but I just want to use this for my own website. I am not insisting that everyone else uses it.

I do think inspectionless team blind is a better event though, because the way it was with inspection, it just turned into 3bld with one competitor telling the other the full memo, and I think it lost most of its value as a team event because of that. Without inspection it seems like this event will have more variety in terms of methods.

**Comment by dmint789** [comment]
I have updated the icons as requested. @jfly is this good to merge now? Preview:

333tbf

**Review by lgarron — CHANGES_REQUESTED** [review]
Sorry for being sporadically active.

I definitely want to see a team BLD icon and I think the two-dot smiley is quite nice. But I still have two concerns about landing:

- I am strongly against an "old" icon for team BLD in this repo at the moment. (Perhaps this can be solved by supporting some transformations outside of assigning icons, like putting a strike through any icon.)
- I would really like to avoid cryptically short event IDs going forward, particularly if they don't match up with what people actually call events — it would be like using 3-letter variable names in programming because they had to do it in the 80's. I'd strongly prefer `333_team_bld` or at least `333_team_bf`.

@jfly, what's your take on these concerns?

**Comment by jfly** [comment]
> I am strongly against an "old" icon for team BLD in this repo at the moment

I agree. I'd prefer to reserve that for events that were once WCA official and are no longer official.

@dmint789, how about we rework this PR to just add one event icon (333_team_bld). It's an unofficial event without regulations, I don't think this repo should be in the business of distinguishing between the various regulations that have been invented for unofficial events.

> I would really like to avoid cryptically short event IDs going forward

It's amusing to me that the icons repo might be the place that sets the standard for these codes in the future! I agree that the 3-4 character codes are overly obtuse. I vote for `333_team_bld`.

**Comment by dmint789** [comment]
We have to at least use kebab-case (333-team-bld) to stay consistent, since 333-mirror-blocks uses kebab-case. @jfly I also REALLY like the idea of cubing icons setting naming standards. You should add a small section about that. I, for one, got confused at first, cause I thought the word "unofficial" was part of the event names. And fine, I'll just use my own icon for old style, you don't need to make the repo confusing due to my website's quirk.

**Comment by dmint789** [comment]
Done

**Review by jfly — APPROVED** [review]

**Review by lgarron — APPROVED** [review]
😀

**Comment by dmint789** [comment]
Is one of you guys supposed to run the workflow or how does that work?

**Comment by jfly** [comment]
> You should add a small section about that. I, for one, got confused at first, cause I thought the word "unofficial" was part of the event names

Yeah, I can see how that would be confusing! Would you be up for adding a blurp about the structure of this repo to the READMe?

**Comment by dmint789** [comment]
> > You should add a small section about that. I, for one, got confused at first, cause I thought the word "unofficial" was part of the event names
> 
> Yeah, I can see how that would be confusing! Would you be up for adding a blurp about the structure of this repo to the READMe?

Sure. You mean the main README of the repo? Is that connected to the website's front page?

**Comment by jfly** [comment]
> Sure. You mean the main README of the repo? Is that connected to the website's front page?

After thinking about this a bit, I think this is a bit of a hairier problem than I realized. Let's continue this discussion over on https://github.com/cubing/icons/issues/108.

**Comment by dmint789** [comment]
Is this PR good to be merged now? Are we gonna stick with kebab-case or switch to snake_case?

**Comment by jfly** [comment]
LGTM! Just merged it =)

**Pull request merged**